specification/resources/resource-manager/Microsoft.Resources/stable/2017-05-10/resources.json

```diff
@@ -34,6 +34,321 @@
     }
   },
   "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Resources/deployments/{deploymentName}": {
+      "delete": {
+        "tags": [
+          "Deployments"
+        ],
+        "operationId": "Deployments_Delete",
```

Comment:
>Deployments_Delete [](start = 24, length = 18)

We should use a different operation Id, so that it doesn't conflict with the existing RG level deployment APIs below